Dashboard/app/js/lib/views/maps/map-views-common.js

```diff
@@ -660,21 +707,39 @@ FLOW.NavMapsView = FLOW.View.extend({
                 if(questionAnswer !== "" && questionAnswer !== null && questionAnswer !== "null"){
                   switch (self.questions[i].questionType) {
                     case "PHOTO":
-                      var imageString = "", imageJson;
+                    case "VIDEO":
+                      var mediaString = "", mediaJson, mediaFilename = "", mediaObject = {};
                       if (questionAnswer.charAt(0) === '{') {
-                        imageJson = JSON.parse(questionAnswer);
-                        imageString = imageJson.filename
+                        mediaJson = JSON.parse(questionAnswer);
+                        mediaString = mediaJson.filename;
+
+                        if(mediaJson.location != null) {
```

Comment:
I would suggest to use here `if (mediaJson.location) {...}` since this would cover more false conditions.